src/cmd/cmdHandler.ts

```diff
@@ -1,18 +1,10 @@
 import * as vscode from 'vscode';
+import * as path from 'path';
 import { ChildProcessWithoutNullStreams, exec, spawn } from "child_process";
-
 import pickExplorerFile from "./pickExplorerFile";
 import { Nanos } from "./NanosRepo";
-
-interface Ops {
```

Comment:
Using an interface instead of the concrete class helps you to test the command handler without calling `ops` commands. Currently, we haven't covered this plugin with tests, but would be interesting/useful to do it for code quality purposes.

Reply:
Although it requires changing the interface every time the class Ops is updated.

Reply:
Fixed by https://github.com/nanovms/ops-vscode/pull/13/commits/8260f4f48c993ad92744decb4746216d30e829f4